source/gui.cpp

```diff
@@ -571,7 +571,7 @@ void reshade::runtime::draw_overlay_menu_home()
 	if (!_effects_enabled)
 		ImGui::Text("Effects are disabled. Press '%s' to enable them again.", input::key_name(_effects_key_data).c_str());
 
-	if (!_screenshot_path_exists && (_screenshot_key_data[0] != 0 || _screenshot_path != ""))
+	if (!_screenshot_path_exists && (_screenshot_key_data[0] != 0 || !_screenshot_path.empty()))
```

Comment:
Now that I think about it, shouldn't this be `if (!_screenshot_path_exists && _screenshot_key_data[0] != 0)`? The path needs to be valid regardless of whether it is empty or not if a screenshot key is bound.

Reply:
Should I change title and code on this pull request?
If apply it idea, more require changes to some other functions.